install/pre-install-payload/containerd/payload.sh

```diff
@@ -42,8 +44,9 @@ function build_containerd_payload() {
 
 		echo "Building containerd payload image for ${arch}"
 		docker buildx build \
-			--build-arg ARCH="${kernel_arch}" \
-			-f "containerd/payload_Dockerfile" \
+			--build-arg ARCH="${golang_arch}" \
+			--build-arg VERSION="${containerd_version}" \
```

Comment:
It would be good to have `--build-arg IMAGE="${image}" \` here.

Reply:
Done!